src/withMeasure.js

```diff
@@ -7,13 +7,27 @@ import { deepElementFromPoint } from "./util";
 let nodeAtPointerRef;
 const pointer = { x: 0, y: 0 };
 
-function findAndDrawElement(x, y) {
+function findAndDrawElement(x, y, tokens) {
   nodeAtPointerRef = deepElementFromPoint(x, y);
-  drawSelectedElement(nodeAtPointerRef);
+  drawSelectedElement(nodeAtPointerRef, tokens);
 }
 
 export const withMeasure = (StoryFn, context) => {
   const { measureEnabled } = context.globals;
+  const {
+    measure: measureParam = {},
+    tokens: tokensParam,
+  } = context.parameters;
```

Comment:
If there's a way to access `parameters` in `src/box-model/tokenize.js`, then we wouldn't have to access it here and pass it all the way through the call stack.

Reply:
There is a `useParameter` hook in Storybook API. You could try that but, I'm not entirely sure if it will work inside `tokenize.js`

Reply:
Thanks, Varun. Admittedly, I didn't try it, but as it's a React hook, I felt confident it would have to be called inside a React function component.

Maybe [`addons.getChannel`](https://storybook.js.org/docs/react/addons/addons-api#addonsgetchannel) would allow referencing `parameters`, but it seemed heavy, as it's designed for two-way communication.